portal/config.py

```diff
@@ -52,6 +52,7 @@ class BaseConfig(object):
     GOOGLE_CONSUMER_SECRET = os.environ.get('GOOGLE_CONSUMER_SECRET', '')
 
     DEFAULT_LOCALE = 'en_US'
+    FILE_UPLOAD_DIR = 'uploads'
```

Comment:
brilliant - now if you could add a different value for FILE_UPLOAD_DIR to the TestConfig (below in same file), you're set.